src/main/webapp/app/entities/file-details.model.ts

```diff
@@ -0,0 +1,18 @@
+export class FileDetails {
+    constructor(
+        public name: string = '',
+        public extension: string = '',
+    ) {}
```

Comment:
```suggestion
    ) {}

```
Nice refactoring of functionality that is used in multiple places!